corehq/toggles.py

```diff
@@ -395,16 +395,13 @@ def toggles_dict(username=None, domain=None):
     [NAMESPACE_USER, NAMESPACE_DOMAIN],
 )
 
-USE_NEW_TIMEZONE_BEHAVIOR = StaticToggle(
-    'use_new_timezone_behavior',
-    ("Enable properly dealing with timezones in phone timestamps "
-     "during submission and in reports. "
-     "(Please do not set manually, "
-     "because it has to be accompanied by a migration.)"),
-    TAG_EXPERIMENTAL,
-    [NAMESPACE_DOMAIN],
+BULK_PAYMENTS = StaticToggle(
+    'bulk_payments',
+    'Enable payment of invoices by bulk credit payments and invoice generation for wire transfers',
+    TAG_PRODUCT_CORE
```

Comment:
Must be. I think there was a conflict. Might be worth rebasing as a way to make sure